Resources/Private/JavaScript/DlfMediaPlayer/Markers.test.js

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+
+import { beforeEach, describe, expect, jest, test } from '@jest/globals';
```

Comment:
```suggestion
import { describe, expect, jest, test } from '@jest/globals';
```
It seems like `beforeEach` is imported but never used.